src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -57,12 +59,11 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
      * Function that writes the value safely.
      * @param value as dayjs or date
      */
-    writeValue(value: any) {
-        // convert dayjs to date, because owl-date-time only works correctly with date objects
-        if (dayjs.isDayjs(value)) {
-            this.value = (value as dayjs.Dayjs).toDate();
-        } else {
-            this.value = value;
+    writeValue(value: dayjs.Dayjs | Date | null): void {
+        if (value !== undefined && this.value !== value) {
+            this.value = !value ? null : isDate(value) ? value : dayjs(value).toDate();
+            this.value?.setSeconds(0, 0);
+            this._onChange(dayjs(value));
```

Comment:
You wrap the value in a dayjs. Meaning if the value is null, `this.value` and the parameter of `this._onChange` will differ.

Is this intended behaviour? If yes, it lacks documentation. Otherwise, it needs adaptation.

Reply:
not needed anymore